src/Route.jsx

```diff
@@ -11,6 +11,12 @@ const ICON_MAPPING = {
   },
```

Comment:
Since we only have bus & stop icons, would it make sense to rename this to `STOP_ICON_MAPPING`?